.devcontainer/requirements.txt

```diff
@@ -17,5 +17,5 @@ ruff==0.4.6
 textalloc==1.1.3
```

Comment:
Hi @JulesL2 
Happy new year :tada: 
I've just merged a significant refactoring that makes the code more modular and allows to change  through the GUI any param we want ! (PR https://github.com/ThomasParistech/pretty-gpx/pull/81)
Conflicts will be terrible to fix, I suggest you to start from a new branch :wink: